PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl.rbxmx

```diff
@@ -19,7 +19,7 @@ local Players = game:GetService('Players')
 local RunService = game:GetService('RunService')
 local UserInputService = game:GetService('UserInputService')
 
-local HasVRAPI = pcall(function() local _ = UserInputService.VREnabled end)
+local HasVRAPI = pcall(function() return UserInputService.VREnabled and UserInputService.UserHeadCFrame end)
```

Comment:
Right. For some reason I was thinking of holding off until it was on all platforms, but that's not necessary.